evaluate/metrics.go

```diff
@@ -16,31 +17,37 @@ type Metrics struct {
 	Total uint
 	// Executed is the number of benchmarking candidates with successful execution.
 	Executed uint
+
+	// Coverage holds the coverage of the benchmarking candidates.
+	Coverage []float64
```

Comment:
Do not use float, because it has problems to keep numbers straight. Use uint for metrics.

Reply:
But coverage reportings from Go have floating point digits. Or should we just cut them off?

Reply:
Lets move it to the TODO. i didn't expect that we are collecting percentage. We should collect the absolute metrics and then do percentage as a summary. Not as a result of the benchmark. That is imprecise the more data we have.